clients/cloud/java/README.md

```diff
@@ -59,70 +60,142 @@ The Kafka Streams API reads the same topic from Confluent Cloud and does a state
+    Consumed record with key alice and value {"count":7}, and updated total count to 28
+    Consumed record with key alice and value {"count":8}, and updated total count to 36
+    Consumed record with key alice and value {"count":9}, and updated total count to 45
+    ```
+    
+    When you are done, press `<ctrl>-c`.
+
+3. Run the Kafka Streams application, passing in arguments for (a) the local file with configuration parameters to connect to your Confluent Cloud instance and (b) the same topic name as used above. Verify that the application received all the messages:
+
+    ```shell
+    # Compile the Java code
+    $ mvn clean package
+
+    # Run the consumer
+    $ mvn exec:java -Dexec.mainClass="io.confluent.examples.clients.cloud.StreamsExample" \
+      -Dexec.args="$HOME/.ccloud/config test1"
+    ```
+
+    You should see:
+
+    ```
+    ...
+    [Consumed record]: alice, 0
+    [Consumed record]: alice, 1
+    [Consumed record]: alice, 2
+    [Consumed record]: alice, 3
+    [Consumed record]: alice, 4
+    [Consumed record]: alice, 5
+    [Consumed record]: alice, 6
+    [Consumed record]: alice, 7
+    [Consumed record]: alice, 8
+    [Consumed record]: alice, 9
+    ...
+    [Running count]: alice, 0
+    [Running count]: alice, 1
+    [Running count]: alice, 3
+    [Running count]: alice, 6
+    [Running count]: alice, 10
+    [Running count]: alice, 15
+    [Running count]: alice, 21
+    [Running count]: alice, 28
+    [Running count]: alice, 36
+    [Running count]: alice, 45
+    ...
+    ```
+        
+    When you are done, press `<ctrl>-c`.
+
+
+# Example 2: Avro And Confluent Cloud Schema Registry
+
+This example is similar to the previous example, except the value is formatted as Avro and integrates with the Confluent Cloud Schema Registry.
+Check availability of Confluent Cloud Schema Registry at https://www.confluent.io/confluent-cloud/ .
+
+1. In the Confluent Cloud GUI, create an API key and secret to Confluent Cloud Schema Registry.
+
+2. Verify your Confluent Cloud Schema Registry credentials work from your host. In the output below, substitute your values for `<SR API KEY>`, `<SR API SECRET>`, and `<SR ENDPOINT>`.
+
+    ```shell
+    # View the list of registered subjects
+    $ curl -u <SR API KEY>:<SR API SECRET> https://<SR ENDPOINT>/subjects
+    ```
+
+3. Add the following parameters to your local Confluent Cloud configuration file. In the output below, substitute values for `<SR API KEY>`, `<SR API SECRET>`, and `<SR ENDPOINT>`.
+
+    ```shell
+    $ cat $HOME/.ccloud/config
+    ...
+    basic.auth.credentials.source=USER_INFO
+    schema.registry.basic.auth.user.info=<SR API KEY>:<SR API SECRET>
+    schema.registry.url=https://<SR ENDPOINT>
+    ...
+    ```
+
+4. Run the Avro producer, passing in arguments for (a) the local file with configuration parameters to connect to your Confluent Cloud instance and (b) the topic name:
```

Comment:
I think I missed something here. Was a topic called test2 created?

Reply:
@NathanNam good question.  I added a comment to the README that explains:

```
    # If the topic does not already exist, the code will use the Kafka Admin Client API to create the topic
```

Reply:
Thanks for adding this part.